src/table/__integ__/resizable-columns.test.ts

```diff
@@ -60,6 +60,20 @@ class TablePage extends BasePageObject {
     return element.getAttribute('style');
   }
 
+  getFirstTableHeaderWidths() {
+    return this.browser.execute(() => {
+      const tables = document.querySelectorAll('table');
```

Comment:
Instead of using `document.querySelectorAll` and picking the first item, we could simplify a little bit here by just using `document.querySelector`